### PR TITLE
doc: use emulator from cli

### PR DESCRIPTION
**Pull request by itsspriyansh**

### CODE REVIEW

File: docs/use-emulator-from-command-line.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+```bash
+# shows a list of available webcams
+emulator @nightwatch-android-11 -webcam-list
+
+# configures front camera of the emulated device
+emulator @nightwatch-android-11 -camera-front mode
+
+# configures back camera of the emulated device
+emulator @nightwatch-android-11 -camera-back mode
```
Comment (garg3133) [inline docs/use-emulator-from-command-line.md:150]:
```suggestion
# configures front camera of the emulated device
emulator @nightwatch-android-11 -camera-front <mode>

# configures back camera of the emulated device
emulator @nightwatch-android-11 -camera-back <mode>
```

**Review by garg3133 — COMMENTED** [review]
This is superb! Thanks a lot!

File: docs/use-emulator-from-command-line.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+To use `emulator` binary directly (without adding it to `PATH`), simply go to the directory where the binary is present and use it as follows:
+
+```bash
+cd /path/to/Android/Sdk/emulator/
+
+#for windows
+emulator --version
+
+#for mac/linux
+./emulator --version
```
[the remark this answers is not in the document]
Comment (itsspriyansh) [reply docs/use-emulator-from-command-line.md:54]: I think in windows executables are callable with just filenames without mentioning `.exe`. However to make it clear for users I think I should still mention `.exe` at the end.

File: docs/use-emulator-from-command-line.md [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+
+### Using `emulator` binary directly
+
+To use `emulator` binary directly (without adding it to `PATH`), simply go to the directory where the binary is present and use it as follows:
+
+```bash
+cd /path/to/Android/Sdk/emulator/
+
+#for windows
+emulator --version
```
[the remark this answers is not in the document]
Comment (itsspriyansh) [reply docs/use-emulator-from-command-line.md:51]: I think in windows executables are callable with just filenames without mentioning `.exe`. However to make it clear for users I think I should still mention `.exe` at the end.

**Review by garg3133 — APPROVED** [review]
LGTM!

**Pull request merged**